rpc/rpc.proto

```diff
@@ -38,7 +38,8 @@ message AddNetworkReply {
   int32 ParentIfIndex = 10;
   // end of pod-eni parameters
 
-  // next field: 13
+  bool EnableNetworkPolicy = 13;
```

Comment:
Is the motivation for this field so that the CNI knows whether to make the NP RPC call?

Reply:
yes similar to SGPP, external SNAT..